src/main/java/flow/Constants.java

```diff
@@ -0,0 +1,20 @@
+package flow;
+
+@SuppressWarnings("javadoc")
+public class Constants {
+
+	protected static final String MAIN_RESOURCES_PATH = "src/main/resources/";
```

Comment:
I would propose to use a more "functional" way of getting the proper path: Thread.currentThread().getContextClassLoader().getResourceAsStream($filename)